### DISCUSSION TITLE
Weather widget borders bugged, cut content

### DISCUSSION THREAD

**Discussion by DRAKONIIUS** [valinet/ExplorerPatcher#3386] [General]
<img width="1180" alt="Screenshot 2024-06-14 153218" src="https://github.com/valinet/ExplorerPatcher/assets/114786355/3826202c-99b7-4af5-9261-41d9d170a8c4">
As I change the corner shape it always displays the not rounded one.
The text in the top border is cut.

**Comment by pyrates999** [comment]
Update to the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Reply by DRAKONIIUS** [reply]
Ok, updated from ...*.65.2 to ...*.65.5 and the text on top of the window has been restored, instead the corners are always displayed not rounded (minor thing).
Thank you.
